rotkehlchen/data_import/utils.py

```diff
@@ -107,7 +107,17 @@ def maybe_flush_all(self, cursor: 'DBCursor') -> None:
 
     def flush_all(self, write_cursor: 'DBCursor') -> None:
         self.db.add_margin_positions(write_cursor, margin_positions=self._margin_trades)
-        self.history_db.add_history_events(write_cursor, history=self._history_events)
+        for event in self._history_events:
+            if self.history_db.add_history_event(write_cursor=write_cursor, event=event) is None:
+                self.append_msg(
+                    row_index=-1,  # we don't know the row index.
+                    msg=(
+                        f'Skipped duplicate {event.entry_type}: {event.event_type.serialize()}/{event.event_subtype.serialize()} '  # noqa: E501
+                        f'of {event.amount} {event.asset.symbol_or_name()} at timestamp {event.timestamp}'  # noqa: E501
```

Comment:
is timestamp given as timestamp in the CSVs? Or as date? If as date, then transform to date here

Reply:
all except our generic importers are in datetime format. adjusted it.